tests/test_marshmallow.py

```diff
@@ -204,6 +234,26 @@ class Bag(Document):
         assert not ret.errors
         assert ret.data == bag.to_mongo()
 
+        data = {
+            'id': {'brief': 'sportbag', 'value': 100, 'name': 'Unknown'},
+            'content': [
+                {'brief': 'cellphone', 'value': 500, 'name': 'Unknown'},
+                {'brief': 'lighter', 'value': 2, 'name': 'Unknown'}]
+        }
+        ret = ma_schema.load(data)
+        assert ret.errors
```

Comment:
`assert ret.errors` seems too vague, could it be possible to test the error dict itself ?

```
assert ret.errors = {'_schema': ['tough shit...']}
# Or at least test the field causing the error
assert '_schema' in ret.errors
```